test/integration/test_coarsening.py

```diff
@@ -39,9 +42,10 @@ def test_coarse_grid_2d( self ):
 
 #------------------------------------------------------------------------------#
 
-    def test_coarse_grid_3d( self ):
+    def test_coarse_grid_3d(self):
         g = structured.CartGrid([2, 2, 2])
-        g = generate_coarse_grid( g, [0, 0, 0, 0, 1, 1, 2, 2] )
+        g.compute_geometry()
+        co.generate_coarse_grid( g, [0, 0, 0, 0, 1, 1, 2, 2] )
```

Comment:
Pylint is not going to like the white spaces in ( g ). Applies to a lot of places.